src/shared/components/lazyMobXTable/LazyMobXTable.tsx

```diff
@@ -134,6 +134,35 @@ export function lazyMobXTableSort<T>(data:T[], metric:SortMetric<T>, ascending:b
     return dataAndValue.map(x=>x.data);
 }
 
+function getListOfEmptyStrings<T>(rowData:T) {
+    if (rowData instanceof Array) {
+        return Array(rowData.length).join(".").split(".");
+    } else {
+        return [""];
+    }
+}
+
+function getAsList<T>(rowData: T, data: string|string[]) {
+    let length = 1;
+    if (data instanceof Array) {
+        if (!(rowData instanceof Array) || rowData.length != data.length) {
+            throw new Error('Unexpected error!');
```

Comment:
@adamabeshouse we can make it a bit more flexible by allowing the output of download to be either 0, 1 or the same length as rowData. But having output of download be e.g. 2 while rowData is 3 would still be an error, right?